diesel_codegen_shared/src/schema_inference/pg.rs

```diff
@@ -74,19 +88,51 @@ mod information_schema {
     }
 }
 
-pub fn determine_column_type(attr: &ColumnInformation) -> Result<ColumnType, Box<Error>> {
+pub fn camel_cased(snake_case: &str) -> String {
```

Comment:
Yes! Finally! PascalCase! Errrr, I mean, "upper camel case". (On a more serious note, I think this `fn` should be called `snake_to_type_case`.)

Please note that all other case conversions we do are trivial conversions by choice, to not introduce surprisingly complex behavior.

The snake case version we get here are valid Postgres identifiers, right? What properties do they have in comparison to Rusts type identifiers?